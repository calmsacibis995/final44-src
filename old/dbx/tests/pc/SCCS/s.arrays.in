h16922
s 00012/00000/00000
d D 5.1 90/06/25 15:36:24 bostic 1 0
c date and time created 90/06/25 15:36:24 by bostic
e
u
U
t
T
I 1
whatis p.a
stop at 12
run
print a
print a[3], a[4], a[5]
step
step
step
step
where
print b, c
print b[BLUE]
E 1
