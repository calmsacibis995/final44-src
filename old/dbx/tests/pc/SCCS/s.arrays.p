h36519
s 00023/00000/00000
d D 5.1 90/06/25 15:36:27 bostic 1 0
c date and time created 90/06/25 15:36:27 by bostic
e
u
U
t
T
I 1
program test (input, output);
type
    Color = (RED, BLUE, GREEN);
    IntArray = array [1..10] of integer;
var a : IntArray;
    i : integer;
    b : array [Color] of integer;
    c : Color;

procedure p (i : integer; var a : IntArray; j : integer);
begin
    a[3] := i;
    a[4] := j;
end;

begin
    for i := 1 to 10 do begin
	a[i] := i;
    end;
    p(4, a, 5);
    b[BLUE] := 3;
    c := RED;
end.
E 1
