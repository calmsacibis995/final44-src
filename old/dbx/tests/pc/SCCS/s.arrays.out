h39647
s 00020/00000/00000
d D 5.1 90/06/25 15:36:26 bostic 1 0
c date and time created 90/06/25 15:36:26 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
(var parameter) a : IntArray;
[1] stop at 12
[1] stopped in p at line 12
   12       a[3] := i;
(1, 2, 3, 4, 5, 6, 7, 8, 9, 10) 
3 4 5 
stopped in p at line 13
   13       a[4] := j;
stopped in p at line 14
   14   end;
stopped in program at line 21
   21       b[BLUE] := 3;
stopped in program at line 22
   22       c := RED;
program(), line 22 in "arrays.p"
main(0x1, 0x7fffe7e4, 0x7fffe7ec) at 0xad
(0, 3, 0) RED 
3 

E 1
